assets/javascripts/discourse-assign/initializers/extend-for-assigns.js.es6

```diff
@@ -153,42 +153,24 @@ function initialize(api) {
     }
   });
 
-  api.createWidget("assigned-to", {
```

Comment:
Wow, why was this widget declared 3 times? 😲

Reply:
ooooops